accessible/field-segment.component.js

```diff
@@ -144,7 +150,12 @@ blocklyApp.FieldSegmentComponent = ng.core.Component({
   // Confirm a selection for dropdown fields.
   selectOption: function() {
     if (this.optionValue == Blockly.Msg.RENAME_VARIABLE) {
```

Comment:
> Short term might be having the dropdown maintain which indices those special values are in and ask each time.

@seanlip And for more context, that's what web blockly does now: it stores the index when it adds the rename option to the list, and compares to that later.  That's what Cory's talking about when he says he'll add a common code path in a later PR.

https://github.com/google/blockly/blob/master/core/field_variable.js#L145